components/raftstore/src/store/fsm/peer.rs

```diff
@@ -392,6 +393,12 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         self.register_split_region_check_tick();
         self.register_check_peer_stale_state_tick();
         self.on_check_merge();
+        // Apply committed entries more quickly.
+        if self.fsm.peer.raft_group.get_store().committed_index()
```

Comment:
Maybe we can do the raft base tick instead?

Reply:
Seems the raft base tick does not generate `Ready` for followers.

Reply:
I find a better place to set has_ready https://github.com/tikv/tikv/blob/7ae2018fe2213282796148af9f125804bf7abf35/components/raftstore/src/store/fsm/peer.rs#L1617.